quiz/small_theme.inc

```diff
@@ -3,6 +3,7 @@ include_once($relPath.'html_page_common.inc'); // output_html_header()
 include_once($relPath.'theme.inc'); // headerbar_text_array()
 include_once($relPath.'faq.inc');
 include_once($relPath.'quizzes.inc'); // get_quiz_id_param get_Quiz_with_id
+include_once($relPath.'Users.inc'); // get_quiz_id_param get_Quiz_with_id
```

Comment:
Should be `User.inc`. (Took me a while -- it was staring me right in the face and I still didn'nt see it!)